camel/toolkits/search_toolkit.py

```diff
@@ -1064,6 +1064,137 @@ def search_exa(
         except Exception as e:
             return {"error": f"Exa search failed: {e!s}"}
 
+    @api_keys_required([(None, 'TONGXIAO_API_KEY')])
+    def search_ali(
```

Comment:
Maybe make the name more specific:
```suggestion
    def search_alibaba_tongxiao(
```